lib/utils/timetable.ts

```diff
@@ -213,3 +213,21 @@ export const camelCaseToKebabCase = (str: string): string => {
 export const kebabCaseToCamelCase = (str: string): string => {
   return str.replace(/-./g, (x) => x[1].toUpperCase());
 };
+
+const dateIsInThePast = (date: string) =>
+  new Date(date).getTime() < new Date().getTime();
```

Comment:
It's not going to make a difference but should this technically be `<=` (I think it's correct as is for the end date)

Reply:
makes sense :) done!